src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,15 +1,45 @@
-# TO-DO: complete the helpe function below to merge 2 sorted arrays
+# TO-DO: complete the helper function below to merge 2 sorted arrays
 def merge( arrA, arrB ):
     elements = len( arrA ) + len( arrB )
     merged_arr = [0] * elements
     # TO-DO
-    
+    # index iterations for each arr
+    a, b, c = 0, 0, 0
+    # compare and sort the elements of both of the arrays to a merged array
+    while a < len(arrA) and b < len(arrB):
```

Comment:
This loop will terminate when you reach the end of either arrA or arrB, so if they aren't the same length, you'll be missing some items--which is why you're needing two extra loops below. How might you be able to iterate over the full length of arrA and arrB? You're incrementing `c` on every loop. maybe you can utilize that?